### ISSUE TITLE
Not working in VUE-3

### ISSUE THREAD

**Issue by sloopsight** [txlabs/vnodes#11]
Please make this lib compatible with vue3.
When I am running this demo on VUE3, we get vnode_WEBPACK_3 not found exception on new `graph()` method.

**Comment by gr1m1um** [comment]
same, using vue3 I was able to get it running by changing how components and graph are imported:

```js
import vnodes from 'vnodes'

export default {
  name: 'App',
  components: {
    Screen: vnodes.Screen,
    Edge: vnodes.Edge,
    Node: vnodes.Node
  },
  data () {
    return {
      graph: new vnodes.graph() // eslint-disable-line
    }
  }
```

also disabling `this.$on` on Node component
```js
if (this.$on) { // vue3 patch
    this.$on('drag', ({ x, y }) => {
    this.data.x += x
    this.data.y += y
  })
}
```

this is for future reference as I am not currently working on a vue3 port.

**Comment by gr1m1um** [comment]
fixed this on latest 1.1.12
